src/vs/workbench/contrib/chat/browser/chatWidget.ts

```diff
@@ -121,6 +124,7 @@ export class ChatWidget extends Disposable implements IChatWidget {
 		this.requestInProgress = CONTEXT_CHAT_REQUEST_IN_PROGRESS.bindTo(contextKeyService);
 
 		this._register((chatWidgetService as ChatWidgetService).register(this));
+		this._register(this.chatService.onDidSubmitSlashCommand(({ slashCommand }) => this.repopulateSlashCommand(slashCommand)));
```

Comment:
Feels wrong to listen to this event when the chat widget knows that it submitted a slash command. Also this would need to check the sessionId